openapi/kas-fleet-manager-private.yaml

```diff
@@ -19,8 +19,56 @@ servers:
 tags:
   - name: Agent Clusters
     description: ""
+  - name: observatorium-proxy
+    description: Observatorium proxy related endpoints
 
 paths:
```

Comment:
The minor version of the api document should have been increased because of this change